package.json

```diff
@@ -22,14 +22,14 @@
     "@mapbox/s3urls": "^1.5.3",
     "@mapbox/tilejson": "^1.0.1",
     "@mapbox/tilelive": "~5.12.2",
-    "@mapbox/tilelive-omnivore": "~4.1.0",
+    "@mapbox/tilelive-omnivore": "^4.2.0",
     "@mapbox/tilelive-s3": "6.5.1",
     "@mapbox/tilelive-vector": "^4.2.0",
     "@mapbox/tiletype": "^0.3.0",
     "@mapbox/vtvalidate": "^0.2.2",
     "mapbox-studio-default-fonts": "https://mapbox-npm.s3.amazonaws.com/package/mapbox-studio-default-fonts-0.0.4-4afb5235f457bd1c1a5a70fce6c2aa83bf7a851e.tgz",
     "mapbox-studio-pro-fonts": "https://mapbox-npm.s3.amazonaws.com/package/mapbox-studio-pro-fonts-1.0.0-9870a90b713f307b9391829602f4d5857e419615.tgz",
-    "mapnik": "~3.7.0",
+    "mapnik": "4.x || 3.x",
```

Comment:
Why the loose versioning here?

Reply:
Matching other patterns that have been established as of late, such as https://github.com/mapbox/tilelive-bridge/blob/40f891f19d215410b2b7510d61aa99d7d634f6ea/package.json#L17